### DISCUSSION TITLE
iiNet VANT6 - has anyone been able to enable bridge mode, but still query modem for info?

### DISCUSSION THREAD

**Discussion by adfhogan** [hack-technicolor/hack-technicolor#208] [Q&A] [answered]
Wondering if there's a way to enable bridge mode on a VANT6, but then still be able to query modem for line sync / SNR etc. details?

**Comment by LuKePicci** [comment, marked as answer]
You can, but you have to abandon the concept of "bridge mode". You need to manually configure some interfaces to be bridged to wan and enable a second vlan interface onto the same port so you can basically connect to times from your other device in order to get bot "bridged" wan path and normal accessibility to gateway management interfaces (SSH, webui, ecc.).

You can certainly do this on Homeware because it fully supports such advanced scenario, but your other device must have the same capability as well, which is to create and configure as many interfaces as you need topping multiple virtual interfaces with as many VLAN IDs as you wish on the same port. In other words, your device must me quite professional to support this.

**Comment by adfhogan** [comment]
Ahh.. quite a bit more difficult than ye olde netgear DG834GT I used yonks back then :) (though I guess that was PPPoE to router, not PPPoA link with DHCP). I think it's theoretically possible on my router (RT-AC5300 running AsusWRT Merlin) to enable tagged and untagged VLANs. Good to know it's achievable tho.. thanks for the info!
